toolchain/cc_toolchain_config.bzl

```diff
@@ -131,7 +131,7 @@ def _impl(ctx):
                 actions = [ACTION_NAMES.c_compile, ACTION_NAMES.cpp_compile],
                 flag_groups = [
                     flag_group(
-                        flags = ["-O3", "-ffunction-sections", "-fdata-sections"],
+                        flags = ["-O3", "-ffunction-sections", "-fdata-sections", "--llvm-lto", "1", "-flto"],
```

Comment:
There is also `--llvm-lto` `2` and `3`. I'm running benchmarks now to see if it makes the difference.

Reply:
Doesn't look like it improves anything: https://github.com/envoyproxy/envoy-wasm/issues/602#issuecomment-668860060.